ethpm/_utils/deployments.py

```diff
@@ -47,7 +47,7 @@ def get_linked_deployments(deployments: Dict[str, Any]) -> Dict[str, Any]:
 
 
 def validate_linked_references(
-    link_deps: Tuple[Tuple[int, bytes], ...], bytecode: HexBytes
+    link_deps: Tuple[Tuple[int, bytes], ...], bytecode: HexStr
```

Comment:
@njgheorghita I am not sure why I had to make this change to appease the linter, but do you see any problems with it?

Reply:
Hmm, I can't tell either off the top of my head, but this change doesn't seem problematic to me. 👍